examples/spring/src/integration-test/kotlin/com/expediagroup/graphql/examples/Constants.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.examples
+
+import org.springframework.http.MediaType
+
+object Constants {
```

Comment:
no need to wrap those constants in an object -> lets rename the class to `IntegrationTestConstants` and have those const values as top level declarations

Reply:
I renamed the class, made the constants as top level declarations, and moved it under `test` directory.